test/e2e/validation_test.go

```diff
@@ -481,12 +481,12 @@ func testServicesConfigMap(t *testing.T) {
 	t.Run("Services ConfigMap existance", func(t *testing.T) {
 		_, err = tc.client.K8s.CoreV1().ConfigMaps(tc.namespace).Get(context.TODO(), servicesConfigMapName,
 			meta.GetOptions{})
-		assert.NoErrorf(t, err, "error ensuring ConfigMap %s exists", servicesConfigMapName)
+		require.NoErrorf(t, err, "error ensuring ConfigMap %s exists", servicesConfigMapName)
 	})
 
 	t.Run("Services ConfigMap re-creation", func(t *testing.T) {
 		err = tc.testServicesCMRegeneration(servicesConfigMapName)
-		assert.NoErrorf(t, err, "error ensuring ConfigMap %s is re-created when deleted", servicesConfigMapName)
+		require.NoErrorf(t, err, "error ensuring ConfigMap %s is re-created when deleted", servicesConfigMapName)
```

Comment:
I will drop these assert --> require changes, not needed

Reply:
You will drop them back to requires?
They are needed as you've changed the context they are being used in, as these are now separate subtests